site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -329,6 +329,10 @@ The following table lists the currently available permissions:
         many circumstances your extension will not need to declare the <code>"tabs"</code> permission to make use of
         these APIs.</td>
     </tr>
+    <tr id="offscreen">
+      <td><code>"offscreen"</code></td>
+      <td>Gives your extension access to the <a href="/docs/extensions/reference/offscreen/">chrome.offscreen</a> API, allowing developers to create off-screen documents to utilize DOM APIs without obtrusively opening new windows or tabs that interrupt the user experience. Off-screen documents are treated as web pages, and do not have the same permissions as the service worker.</td>
```

Comment:
I only have two pieces of concrete guidance. This clearly takes more than one sentence to explain so let it. Is the list of accessible namespaces short enough to include here? I need to know it either way. We should probably open a separate PR to add this information to the API reference.

Reply:
I'm fairly sure the only API that can be used is chrome.runtime, per https://docs.google.com/document/d/1b-I-vXq2h7OFFmus78jZXIWcKilKJLKLeGplnY9wt7k/edit#heading=h.6pt6jd60gybh (see "Limited extension API access"). We could say something like "Cannot access all extension APIs but can use chrome.runtime" since that avoids a claim that there is literally nothing else?